### PR TITLE
Add comma_last to BibTexWriter.

### LINKED ISSUE #206: ENH: Optional last comma in BibTexWriter
BibTeX allows the comma to be optional at the end of the last field in an entry. Currently, `BibTexWriter` only writes entries with this comma missing, but I would like to have the option of including it.  For example, both of these entries are valid:
```
@ARTICLE{Cesar2013,
  author = {Jean Cesar},
  title = {An amazing title}
}
@ARTICLE{Cesar2013,
  author = {Jean Cesar},
  title = {An amazing title},
}
```
Including the ending comma has the advantage that if I later manually add a new field at the end, I don't have to modify the previous line. This also makes for cleaner file diffs.

I am thinking of something as simple as adding `self.comma_last=False` to `BibTexWriter.__init__`, and modifying [the ending line of `BibTexWriter._entry_to_bibtex`](https://github.com/sciunto-org/python-bibtexparser/blob/f5b1bf8cceeba88367c9e1598133df83b60d80c7/bibtexparser/bwriter.py#L139) to be:

```python
if self.comma_last:
    bibtex += ",\n}\n" + self.entry_separator
else:
    bibtex += "\n}\n" + self.entry_separator
```
Thanks.

### PR DESCRIPTION
**Pull request by u55**
Closes #206.

### CODE REVIEW

**Comment by sciunto** [comment]
Great! Many thanks. I let @omangin merging this.

File: bibtexparser/bwriter.py [hunk 1]
```diff
@@ -68,8 +68,11 @@ def __init__(self, write_common_strings=False):
         self.display_order = []
         #: BibTeX syntax allows comma first syntax
         #: (common in functional languages), use this to enable
-        #: comma first syntax as the bwritter output
+        #: comma first syntax as the bwriter output
         self.comma_first = False
+        #: BibTeX syntax allows the comma to be optional at the end of the last field in an entry.
+        #: Use this to enable writing this last comma in the bwriter output. Defaults: False.
+        self.comma_last = False
```
Comment (mwatts15) [inline bibtexparser/bwriter.py]: My 2p: `comma_last` implies a symmetry with `comma_first` although this is a completely different feature. Something like `add_trailing_comma` would be less confusing.

Comment (u55) [reply bibtexparser/bwriter.py]: `add_trailing_comma` would be fine with me. Maybe I'll wait for input from @sciunto and @omangin before modifying the pull request. Do I need to rebase or submit a different PR?

Comment (omangin) [reply bibtexparser/bwriter.py]: I agree with changing the name. Please do and I will merge just after. (And sorry for the long delay!)

File: bibtexparser/bwriter.py [hunk 2]
```diff
@@ -136,6 +139,7 @@ def _entry_to_bibtex(self, entry):
             except TypeError:
                 raise TypeError(u"The field %s in entry %s must be a string"
                                 % (field, entry['ID']))
+        bibtex += "," if self.comma_last else ""
```
Comment (omangin) [inline bibtexparser/bwriter.py]: When `comma_first` is set, it would make more sense to add `"\n,"` instead.

**Comment by u55** [comment]
Closing this pull request in favor of the rebased pull request #219.

**Pull request closed without merging**